src/ctap/mod.rs

```diff
@@ -532,7 +532,7 @@ impl CtapState {
     ///
     /// This function contains the logic of `parse_command`, minus all CBOR encoding and decoding.
     /// It should make command parsing easier to test.
-    fn process_parsed_command(
+    pub fn process_parsed_command(
         &mut self,
```

Comment:
If we don't do this, we have to use this [crate](https://docs.rs/visibility/latest/visibility/attr.make.html) or make something similar ourselves to achieve conditional visibility. We might want to eventually do it otherwise we'll run into this problem again when fuzzing other private functions to increase coverage.

Reply:
Oh so ctap module is private except when testing, thus we can make public all methods we want to test. Thanks.